### PR TITLE
T : Added missing letter in the test case

### PR DESCRIPTION
**Pull request by KevinKipkorir**
Added missing letter in the test cases

### CODE REVIEW

**Comment by Martchus** [comment]
Can you rephrase the commit message as it has already been suggested (https://github.com/os-autoinst/os-autoinst/pull/1691#issuecomment-861638559)?

(`git commit --amend` and `git push -f …` should do it.)

**Comment by okurz** [comment]
As an exercise I suggest you try to "repair" https://github.com/os-autoinst/os-autoinst/pull/1691 again as well as in the current PR reference https://github.com/os-autoinst/os-autoinst/pull/1691 and explain why you created another PR

**Comment by Martchus** [comment]
The other PR was against the `master` branch on his fork and I don't think one can amend the branch retrospectively. Likely he realized that it is not good practice to use that branch so he created a new PR. Hence I wouldn't say he should repair the other PR.

**Comment by KevinKipkorir** [comment]
So should i just amend and leave it at that?

**Comment by Martchus** [comment]
Yes, please amend the commit message for *this* PR (as explained in https://github.com/os-autoinst/os-autoinst/pull/1691#issuecomment-861638559).

**Review by perlpunk — APPROVED** [review]

**Review by kalikiana — APPROVED** [review]

**Comment by perlpunk** [comment]
A failing test on OBS (openSUSE_Factory):
```
[  126s] 3: [14:05:51] ./10-test-image-conversion-benchmark.t ... ok     2446 ms ( 0.02 usr  0.00 sys +  2.78 cusr  0.35 csys =  3.15 CPU)
[  186s] 3: Timeout for pipe other side helper at ./10-virtio_terminal.t line 33.
[  186s] 3: 
[  186s] 3:     #   Failed test 'no (unexpected) warnings (via END block)'
[  186s] 3:     #   at /usr/lib/perl5/5.32.1/Test/Builder.pm line 153.
[  186s] 3:     # Got the following unexpected warnings:
[  186s] 3:     #   1: Timeout for pipe other side helper at ./10-virtio_terminal.t line 33.
[  186s] 3: [33m[0m[33m[0m[33m[0m
[  186s] 3:     #   Failed test '[snap1] virtio_terminal stored all available data'
[  186s] 3:     #   at ./10-virtio_terminal.t line 136.
[  186s] 3:     #          got: ''
[  186s] 3:     #     expected: 'Test data foo'
[  186s] 3: 
[  186s] 3:     #   Failed test '[snap1] Snapshots data verified'
[  186s] 3:     #   at ./10-virtio_terminal.t line 139.
[  186s] 3:     #     Structures begin differing at:
[  186s] 3:     #          $got->{buffer} = ''
[  186s] 3:     #     $expected->{buffer} = 'Test data foo'
[  186s] 3: [33m[0m[33m[0m[33m[0m[33m[0m[33m[0m
[  186s] 3:     #   Failed test '[snap1] carry over buffer sucessful loaded'
[  186s] 3:     #   at ./10-virtio_terminal.t line 153.
[  186s] 3:     #          got: ''
[  186s] 3:     #     expected: 'Test data foo'
[  186s] 3: [33m[0m    # Looks like you failed 3 tests of 22.
[  186s] 3: 
[  186s] 3: #   Failed test 'Test snapshot handling'
[  186s] 3: #   at ./10-virtio_terminal.t line 169.
[  186s] 3: # Looks like you failed 1 test of 3.
[  186s] 3: [14:06:51] ./10-virtio_terminal.t ................... 
[  186s] 3: Dubious, test returned 1 (wstat 256, 0x100)
[  186s] 3: Failed 1/3 subtests 
```
I think I've seen that before somewhere.
I'll retrigger.

**Comment by okurz** [comment]
@perlpunk I have also seen that in before. See https://github.com/os-autoinst/os-autoinst/issues/1686 which I have created about that
